pages/windows/cipher.md

```diff
@@ -3,14 +3,10 @@
 > Encrypt or decrypt files on NTFS drives.
 > More information: <https://docs.microsoft.com/windows-server/administration/windows-commands/cipher>.
 
-- Encrypt a file or directory:
+- [E]ncrypt/[d]ecrypt the file or directory:
```

Comment:
```suggestion
- [e]ncrypt/[d]ecrypt the file or directory:
```

Reply:
I am not sure about this change because, I don't know whether options are case-sensitive here. If they are case insensitive I suggest don't respect letter case in `[]` to better follow grammar.

Reply:
If the highlighted letter case doesn't match with the option in the example, the highlight is not clear at all. The tldr's users are not expert on tldr's doc rules.

Reply:
I need more opinions to understand what we prefer:

- `[e]ncrypt/[d]ecrypt the file or directory:`
- `Encrypt/[d]ecrypt the file or directory:`

Reply:
Resolved as: `Encrypt/[d]ecrypt the file or directory`.